test/test.config.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) 2012-2019 Digital Bazaar, Inc. All rights reserved.
```

Comment:
sorry, still more in ancillary files.

Reply:
Addressed: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/ba56da9231789067b9e3b3c26ae560a452a64c85